CHANGELOG.next.asciidoc

```diff
@@ -20,6 +20,7 @@ https://github.com/elastic/beats/compare/v6.6.0...6.x[Check the HEAD diff]
 
 - Allow beats to blacklist certain part of the configuration while using Central Management. {pull}9099[9099]
 - Filesets with multiple ingest pipelines added in {pull}8914[8914] only work with Elasticsearch >= 6.5.0 {pull}10001[10001]
+- Ingesting structured Elasticsearch audit logs is only supported with Elasticsearch 6.5.0 and above {pull}8852[8852]
```

Comment:
Ingesting any audit logs is only supported with 6.5 and above with this version, correct?

Reply:
Ah yes, true. Fixing...